StyleCop.Analyzers/StyleCop.Analyzers.Test/LayoutRules/SA1500UnitTests.StatementBlocks.cs

```diff
@@ -0,0 +1,138 @@
+﻿namespace StyleCop.Analyzers.Test.LayoutRules
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using StyleCop.Analyzers.LayoutRules;
+    using TestHelper;
+    using Xunit;
+    using System.Collections.Generic;
+
+
+    /// <summary>
+    /// Unit tests for <see cref="SA1500CurlyBracketsForMultiLineStatementsMustNotShareLine"/>.
+    /// </summary>
+    public partial class SA1500UnitTests : DiagnosticVerifier
+    {
+        public static IEnumerable<object[]> StatementBlocksTokenList
```

Comment:
Couldnt you include fixed and foreach in here and remove the special tests for them?